### DISCUSSION TITLE
玩遊戲順便練習JavaScript ~

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#261] [General]
[https://game.udn.com/game/story/122089/6005535?utm_source=Facebook&utm_medium=game&fbclid=IwAR3hT9oWFg2_ERhmA-6PVTff7kltF-NJw3Uw-znrF5rgPIgVLlFsVrLuweI]() 

Bitburner 這款遊戲一推出就大受好評，在一開始的前置作業必須用JavaScript 讓遊戲自動化，有興趣的同學可以試玩看看～

**Comment by enoch-lightning** [comment]
我點你附的連接還是連回這個頁面，你是不是附網路連結的語法打錯？你是打`[這邊放想要顯示出來的文字](網路連結，就是URL)`嗎？
